elastic/tests/conftest.py

```diff
@@ -8,18 +8,14 @@
 import pytest
 import requests
 
-from .common import HERE, URL, get_es_version
+from .common import HERE, URL
 
 
 @pytest.fixture(scope="session", autouse=True)
 def spin_up_elastic():
     args = [
-        'docker-compose', '-f', os.path.join(HERE, 'compose', 'elastic.yaml')
+        'docker-compose', '-f', os.path.join(HERE, 'compose', 'docker-compose.yaml')
```

Comment:
Just a nit, but if we are changing the name of the compose file to `docker-compose.yaml` any reason to not make this just ` docker-compose', os.path.join(HERE, 'compose')`

Reply:
That would require a `cwd` into the `compose` folder to work

Reply:
You're right, missed that 👍